lib/gamnit/virtual_gamepad/virtual_gamepad_spritesheet.nit

```diff
@@ -0,0 +1,33 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# File generated by svg_to_png_and_nit, do not modify, redef instead
```

Comment:
Yeah, it should not change often and it would otherwise be complicated to find and generate for clients.